platform/platform-paxweb-jettyconfig/src/main/java/org/codice/ddf/pax/web/jetty/ClientInfoFilter.java

```diff
@@ -86,6 +86,13 @@ public void doFilter(
       clientInfoMap.put(SERVLET_CONTEXT_PATH, servletContext.getContextPath());
     }
 
+    clientInfoMap.put("serverName", servletRequest.getServerName());
```

Comment:
✏️ Add the scheme as well

Reply:
The scheme is one of the ones that already existed originally.